docs/migration-guide.md

```diff
@@ -58,171 +82,62 @@ To add Storybook to a project that isn’t currently using Storybook:
 This will:
 
 1. Figure out which renderer (React, Vue, Angular, Web Components), builder (Webpack, Vite), or meta-framework (Next.js, SvelteKit) you’re using
-2. Install Storybook 7 and auto-configure it to mirror project settings
+2. Install Storybook 8 and auto-configure it to mirror project settings
 
 ## Manual migrations
 
-In addition to the automated upgrades above, there are manual migrations that might be required to get Storybook 7 working in your project. We’ve tried to minimize this list to make it easier to upgrade. These include:
-
-- Upgrade MDX1 to MDX2
-- `storiesOf` support discontinued by default
+In addition to the automated upgrades above, there are manual migrations that might be required to get Storybook 8 working in your project. We’ve tried to minimize this list to make it easier to upgrade. These include:
 
-### Upgrade MDX1 to MDX2
+### `storiesOf` to CSF
 
-Storybook 7 uses MDX2 by default for rendering [docs](./writing-docs/index.md). The upgrade from MDX1 to MDX2 is not fully automated, due to the large number of changes between versions. Fortunately, we have some tips to help make it a lot easier.
+Storybook architecture is focused on performance and now needs code that is statically analyzable. For that reason, it does not work with `storiesOf`. We provide a codemod which, in most cases, should automatically make the code changes for you (make sure to update the glob to fit your files):
 
-#### Automatically detect MDX2 errors with a CLI tool
+```sh
+# Convert storiesOf to CSF
```

Comment:
@kylegach can you remove the comments in the shell commands or else if a user copies the snippet it will also copy it as well and if they paste it in a terminal it can cause issues.

Reply:
As of the docs refresh in November, the first line comment is now pulled out of the snippet and used as a title. So clicking the copy button only copies the command itself for this.

![](https://github.com/storybookjs/storybook/assets/486540/a14ab27c-7624-4e44-a0ae-ef88395839fe)